pkg/utils/utils.go

```diff
@@ -170,6 +173,27 @@ func GetTrimmedGroupName(group string) string {
 	return group
 }
 
+// GetRestorePvcSizePercentage - get the percentage of size that need to increased during the kdmp restore PVC creation.
+// In the case failure, retunring zero to assume default 10% value.
+func GetRestorePvcSizePercentage() int {
+	fn := "GetRestorePvcSizePercentage"
+	kdmpData, err := core.Instance().GetConfigMap(drivers.KdmpConfigmapName, drivers.KdmpConfigmapNamespace)
+	if err != nil {
+		logrus.Debugf("%v: error in reading configMap [%v/%v]",
+			fn, drivers.KdmpConfigmapName, drivers.KdmpConfigmapNamespace)
+		return 0
+	}
+	if len(kdmpData.Data[kdmpRestorePvcSizePercentageKey]) != 0 {
+		sizePercentage, err := strconv.Atoi(kdmpData.Data[kdmpRestorePvcSizePercentageKey])
+		if err != nil {
+			logrus.Debugf("%v: error in coverting kdmpRestorePvcSizePercetage to integer: %v", fn, err)
```

Comment:
typo: `coverting` and `kdmpRestorePvcSizePercetage`

Reply:
fixed